src/index.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+const state = {
+  tempValue: 75,
+  color: 'orange',
+  landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+  skyscape: '',
+  background: "url('../assets/summer.jpg')",
+  city: 'Orlando',
+  lat: '',
+  lon: '',
+};
+
+// API calls
+const getLatLon = () => {
+  const inputCityElement = document.getElementById('inputCity').value;
+  axios
+    .get('http://127.0.0.1:5000/location', { params: { q: state.city } })
+    .then((response) => {
+      state.lat = response.data[0].lat;
+      state.lon = response.data[0].lon;
+      console.log(state.lat);
+      console.log(state.lon);
+      getRealTemp();
+    })
+    .catch((error) => {
+      console.log('Error!', error);
+    });
+};
+
+const getRealTemp = () => {
+  console.log(state.lat);
+  console.log(state.lon);
+  axios
+    .get('http://127.0.0.1:5000/weather', {
+      params: {
+        lat: state.lat,
+        lon: state.lon,
+      },
+    })
+    .then((response) => {
+      console.log(response);
+      // const tempValueK = response.data.current.temp;
+      const tempValueK = response.data.current.temp;
+      const tempValueF = ((tempValueK - 273.15) * 9) / 5 + 32;
+      const tempF = Math.floor(tempValueF);
+      state.tempF = tempF;
+      displayRealTemp(tempF);
+      landscapeChangeAction();
+      tempValueColorAction();
+    })
+    .catch((error) => {
+      console.log('Error!', error);
+    });
+};
+
+const displayRealTemp = (tempF) => {
+  const tempValueElement = document.getElementById('tempValue');
+  // tempValueElement.textContent = state.tempF;
+  tempValueElement.textContent = tempF;
+};
+
+// Function to increase temp Value by clicking up arrow
+const incTempAction = () => {
+  state.tempValue += 1;
+  const tempDigit = document.getElementById('tempValue');
+  tempDigit.textContent = state.tempValue;
+  tempValueColorAction();
+  landscapeChangeAction();
+};
+
+// Function to decrease temp Value by clicking up arrow
+const decTempAction = () => {
+  state.tempValue -= 1;
+  const tempDigit = document.getElementById('tempValue');
+  tempDigit.textContent = state.tempValue;
+  tempValueColorAction();
+  landscapeChangeAction();
```

Comment:
Notice how lines 75-78 are the same as lines 66-69. You can put this logic into a helper method and call it in inc/decTempAction methods